api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -478,27 +485,92 @@ public void getSimilarPeople_shouldMatchSingleSearchToAnyNamePart() throws Excep
 	@Test
 	public void getSimilarPeople_shouldMatchTwoWordSearchToAnyNamePart() throws Exception {
 		executeDataSet("org/openmrs/api/include/PersonServiceTest-names.xml");
+		updateSearchIndex();
+		
 		Set<Person> matches = Context.getPersonService().getSimilarPeople("Darius Graham", 1979, "M");
-		Assert.assertEquals(6, matches.size());
+		Assert.assertEquals(11, matches.size());
 		assertTrue(containsId(matches, 1000));
 		assertTrue(containsId(matches, 1003));
-		assertTrue(containsId(matches, 1004));
-		assertTrue(containsId(matches, 1005));
+		assertTrue(containsId(matches, 1004)); 
+		assertTrue(containsId(matches, 1005)); 
 		assertTrue(containsId(matches, 1006));
 		assertTrue(containsId(matches, 1007));
+		
+		assertTrue(containsId(matches, 1009));
+		assertTrue(containsId(matches, 1010));
+		assertTrue(containsId(matches, 1011));
+		assertTrue(containsId(matches, 1012));
+		assertTrue(containsId(matches, 1013));
+		
+		
```

Comment:
I'm not particularly a fan of having a blank line after the last executable statement in a function, but we definitely shouldn't have two.